idaes_examples/notebooks/docs/unit_models/reactors/plug_flow_reactor_usr.ipynb

```diff
@@ -1 +1 @@
  273.15\n", "    Pressure                                          pascal 1.0000e+05 1.0000e+05\n", "====================================================================================\n"]}], "source": ["print(f\"operating cost = ${value(m.fs.operating_cost)/1e6:0.3f} million per year\")\n", "\n", "print()\n", "print(\"Heater results\")\n", "\n", "m.fs.H101.report()\n", "\n", "print()\n", "print(\"PFR reactor results\")\n", "\n", "m.fs.R101.report()"]}, {"cell_type": "markdown", "metadata": {}, "source": ["Display optimal values for the decision variables and design variables:"]}, {"cell_type": "code", "execution_count": 34, "metadata": {}, "outputs": [{"name": "stdout", "output_type": "stream", "text": ["Optimal Values\n", "\n", "H101 outlet temperature = 328.150 K\n", "\n", "Total heat duty required =  -3.440 MJ\n", "\n", "Tube area required = 2.087 m^2\n", "\n", "Tube length required = 4.979 m\n", "\n", "Assuming a 20% design factor for reactor volume,total CSTR volume required = 12.469 m^3 = 3294.093 gal\n", "\n", "Ethylene glycol produced = 225.415 MM lb/year\n", "\n", "Conversion achieved = 90.0%\n"]}], "source": ["print(\"Optimal Values\")\n", "print()\n", "\n", "print(f\"H101 outlet temperature = {value(m.fs.H101.outlet.temperature[0]):0.3f} K\")\n", "\n", "print()\n", "print(\n", "    \"Total heat duty required = \",\n", "    f\"\"\"{(value(m.fs.R101.length) / value(m.fs.R101.config.finite_elements) * (value(sum(m.fs.R101.heat_duty[0, k] for k in  m.fs.R101.control_volume.length_domain if 0.0 <= k < 1.0))\n", "    + (value(m.fs.R101.heat_duty[0, m.fs.R101.control_volume.length_domain.at(1)])\n", "    + value(m.fs.R101.heat_duty[0, m.fs.R101.control_volume.length_domain.at(-1)]))/2))/1e6:0.3f}\"\"\"\n", "    f\" MJ\",\n", ")\n", "print()\n", "print(f\"Tube area required = {value(m.fs.R101.area):0.3f} m^2\")\n", "\n", "print()\n", "print(f\"Tube length required = {value(m.fs.R101.length):0.3f} m\")\n", "\n", "print()\n", "print(\n", "    f\"Assuming a 20% design factor for reactor volume,\"\n", "    f\"total CSTR volume required = {value(1.2*m.fs.R101.volume):0.3f}\"\n", "    f\" m^3 = {value(pyunits.convert(1.2*m.fs.R101.volume, to_units=pyunits.gal)):0.3f} gal\"\n", ")\n", "\n", "print()\n", "print(f\"Ethylene glycol produced = {value(m.fs.eg_prod):0.3f} MM lb/year\")\n", "\n", "print()\n", "print(f\"Conversion achieved = {value(m.fs.R101.conversion):.1%}\")"]}, {"cell_type": "code", "execution_count": null, "metadata": {}, "outputs": [], "source": []}, {"cell_type": "code", "execution_count": null, "metadata": {}, "outputs": [], "source": []}], "metadata": {"celltoolbar": "Tags", "kernelspec": {"display_name": "Python 3 (ipykernel)", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.8.16"}}, "nbformat": 4, "nbformat_minor": 3}
```

Comment:
I am not sure why, but GitHub is rendering these changes in this notebook (and a few others) as a single formatted block of text which makes it next to impossible to review. I think it is in part because the cells were rerun (as they should) meaning a lot of changes were made.

Could you comment on what was changed in these notebooks just so we can be sure - I was surprised to see a reactor example in here.

Reply:
For the four surrogates notebooks, I removed everything having to do with the private method as we discussed on Thursday.

Then, for the PFR and ALAMO examples, the test files needed to be updated. They were somehow skipped while I was debugging the `idaesx` behavior, preprocessing them after resaving the source resolved the test failures.